app/javascript/__tests__/layouts/AssistanceLayout.test.tsx

```diff
@@ -5,6 +5,15 @@ import { renderAndLoadAsync } from "../__util__/renderUtils"
 
 const CHILD_CONTENT = "Content!"
 
+jest.mock("react-helmet-async", () => {
+  return {
+    // eslint-disable-next-line react/prop-types
+    HelmetProvider: ({ children }) => <div>{children}</div>, // Mock HelmetProvider
```

Comment:
Have you considered `HelmetProvider: ({ children }: { children: React.ReactNode })` instead of disabling the linter? `ReactNode` is a very permissive type, so I'm not sure if it is the preferred way to type this.